DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python
-########################################################################
-# $HeadURL$
-########################################################################
-__RCSID__ = "$Id$"
+
+import sys
+
+from DIRAC import gConfig, exit as dexit
```

Comment:
Is it really save (for all eternity) to put this gConfig import before Script.parseCommandLine is called?
(Is there a rule for what can be imported before parseCommandLine ?)